buildchain/make_build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+# Move to source code folder
+cd /home/espressif/ESP32CommandStation
+
+# Config if required
+if [ "x$1" == "xmenuconfig" ]; then
+    idf.py menuconfig
+fi 
+
+# Build
```

Comment:
before building we should call `idf.py set-target {target}` which can be passed in similar to the menuconfig option above. The default target should be esp32 but can also be esp32s3.